modules/deploying-lvms-on-sno-cluster.adoc

```diff
@@ -34,4 +34,43 @@ Before you deploy {lvms} on {sno} clusters, be aware of the following limitation
 
 * You can only create a single instance of the `LVMCluster` custom resource (CR) on an {product-title} cluster.
 * You can make only a single `deviceClass` entry in the `LVMCluster` CR.
-* When a device becomes part of the `LVMCluster` CR, it cannot be removed.
\ No newline at end of file
+* When a device becomes part of the `LVMCluster` CR, it cannot be removed.
+
+[id="lvms-deployment-limitations-for-sno-ran_{context}"]
+== Limitations
+
+For deploying {sno}, LVM Storage has the following limitations:
+
+* The total storage size is limited by the size of the underlying Logical Volume Manager (LVM) thin pool and the overprovisioning factor.
+* The size of the logical volume depends on the size of the Physical Extend (PE) and the Logical Extend (LE).
```

Comment:
@suleymanakbas91 Correct me if wrong, over the internet the 'E' in PE & LE stands for 'Extent'. Not sure if 'Extent' and 'Extend' words are used interchangeably?

Reply:
Wow, that's a great catch! You're absolutely right. It should be 'Extent' not 'Extend'.

Reply:
Good catch, updated! I'm wondering if it should be corrected in this KB article as well? https://access.redhat.com/solutions/65201

Reply:
I'd say yes.